apis/r/tests/testthat/test-SCEOutgest.R

```diff
@@ -26,7 +26,8 @@ test_that("Load SCE object from ExperimentQuery mechanics", {
     experiment = experiment,
     measurement_name = "RNA"
   )
-  expect_warning(obj <- query$to_single_cell_experiment())
+  #####expect_warning(obj <- query$to_single_cell_experiment())
+  obj <- query$to_single_cell_experiment()
```

Comment:
Is there a reason we're not getting a warning here? I'm intentionally throwing a warning here because of the dense ICA obsm array, so we absolutely should be checking to ensure that warning is coming through. If it's not, then something is broken in the code

Reply:
Oops sorry that was just debug -- let me restore that -- thanks for catching that ...

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/3176/commits/c1b334b435a95521a739ca907322ac5c73a3992a